detail.py

```diff
@@ -0,0 +1,47 @@
+from django_mako_plus import view_function, jscontext
+from catalog import models as cmod
+
+@view_function
+
+def process_request(request, product:cmod.Product):
+
```

Comment:
Be sure to add the ''' DEF DESCRIPTION ''' for your code.

Reply:
will do, thank you!